app/src/routes/project/Canvas/index.tsx

```diff
@@ -62,10 +65,81 @@ const Canvas = ({ children }: { children: ReactNode }) => {
         const div = containerRef.current;
         if (div) {
             div.addEventListener('wheel', handleWheel, { passive: false });
-            return () => div.removeEventListener('wheel', handleWheel);
+            div.addEventListener('mousedown', middleMouseButtonDown);
```

Comment:
I move these from the PanOverlay so that this works on the webview as well, not just the area outside the webview